REFERENCE.md

```diff
@@ -82,75 +82,75 @@ The following parameters are available in the `redis` class.
 
 ##### `activerehashing`
 
-Data type: `String`
```

Comment:
Was this previously documented as a string purely because of explicit puppet-strings tags?  ie it looks like the default was always an actual Boolean.

Reply:
`String` looks a buggy declarative type in doc where `params.pp` use Boolean :
https://github.com/voxpupuli/puppet-redis/pull/328/files#diff-2f383a197b1a428561b364755701d60eL10

Reply:
Correct. Previously it was set via puppet-strings but it really was a boolean. https://github.com/voxpupuli/puppet-redis/blob/c0d0f76a90abebd197f5410df0ed5764b9a6a0aa/templates/redis.conf.erb#L711 makes this very clear.